apps/storefront-e2e/src/integration/category-navigation.cy.ts

```diff
@@ -0,0 +1,40 @@
+import { CategoryNavigationFragment } from '../support/page-fragments/category-navigation.fragment';
+import { LandingPage } from '../support/page-objects/landing.page';
+
+describe('Category navigation suite', () => {
+  let landingPage;
+
+  describe('when the landing page is visited', () => {
+    beforeEach(() => {
+      landingPage = new LandingPage();
+      landingPage.visit();
+    });
+
+    it('should render the category navigation in the header ', () => {
```

Comment:
We can combine these tests in one to make them more cost-efficient ---
And the second check covers the first one, because if the navigation is not rendered - the link text will not be found and both tests will fail. 

So, the first check can be removed.